src/palace/manager/sqlalchemy/model/licensing.py

```diff
@@ -1041,12 +1041,6 @@ def loan_to(
             create_method_kwargs=kwargs,
         )
 
-        if is_new:
-            # This action creates uncertainty about what the patron's
-            # loan activity actually is. We'll need to sync with the
-            # vendor APIs.
-            patron.last_loan_activity_sync = None
-
```

Comment:
Instead of doing this here, we queue up an async task in the loans controller when we make a new loan or hold.